likecoin/public/likecoin.php

```diff
@@ -43,7 +43,7 @@ function likecoin_content_filter( $content ) {
 		if ( ! empty( $post->post_password ) ) {
 			return $content;
 		}
-		$content = $content . likecoin_add_likebutton() . likecoin_add_iscn_badge( $post );
+		$content = $content . likecoin_add_iscn_badge( $post ) . likecoin_add_likebutton() ;
```

Comment:
should hide LikeCoin button?

Reply:
Yes, the LikeCoin button isn't directly related to the project. Showing another UI element would require extra time to explain, so we decided to hide it

Reply:
Yes, the LikeCoin button isn't directly related to the whole project. Adding another UI element would require extra time to explain, so we decided to hide it.

But we'll still keep WNFT widget (for business model KPI ), and only hide the clapping bar.